### DISCUSSION TITLE
how does it work

### DISCUSSION THREAD

**Discussion by spacedrill710** [3kh0/ext-remover#524] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
hi i need to all this and i wondering how this works cause i disabled them and yet nothing happens or changes
![Screenshot 2023-02-15 11 53 31 AM](https://user-images.githubusercontent.com/125504641/219138490-1ea5f5e2-c72b-4b48-8591-f95c1353aa1e.png)
![Screenshot 2023-02-15 11 53 16 AM](https://user-images.githubusercontent.com/125504641/219138494-529807e3-8d30-468b-97bf-6bc44f00274d.png)

^ 
as seen and it does nothing and effect nothing am i doing something wrong??

**Comment by obkduishinijf** [comment, marked as answer]
check chrome://version
if it's v102+ then it won't work
(mark as answer unless this didn't help)

**Reply by spacedrill710** [reply]
is there a way to bypass it? 
![Screenshot 2023-02-15 12 04 13 PM](https://user-images.githubusercontent.com/125504641/219140737-b9348918-51c9-483c-92dc-41442e7e3be0.png)

**Reply by qDread-dev** [reply]
there are some methods that work for later versions (look at the readme)

**Reply by spacedrill710** [reply]
> there are some methods that work for later versions (look at the readme)

okay but i upgrade it back when i dont need it?

**Reply by obkduishinijf** [reply]
> > there are some methods that work for later versions (look at the readme)
> 
> okay but i upgrade it back when i dont need it?

if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)

**Reply by spacedrill710** [reply]
> > > there are some methods that work for later versions (look at the readme)
> > 
> > 
> > okay but i upgrade it back when i dont need it?
> 
> if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)

is it worth it?

**Reply by obkduishinijf** [reply]
> > > > there are some methods that work for later versions (look at the readme)
> > > 
> > > 
> > > okay but i upgrade it back when i dont need it?
> > 
> > 
> > if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)
> 
> is it worth it?

not really, if you get caught you can be permanently banned from all school-issued devices, as it goes on your permanent record.

**Reply by spacedrill710** [reply]
> > > > > there are some methods that work for later versions (look at the readme)
> > > > 
> > > > 
> > > > okay but i upgrade it back when i dont need it?
> > > 
> > > 
> > > if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)
> > 
> > 
> > is it worth it?
> 
> not really, if you get caught you can be permanently banned from all school-issued devices, as it goes on your permanent record.

so is there a less risk way that you can tell me thats better then downgrading i am looking at the readme thing

**Reply by obkduishinijf** [reply]
> > > > > > there are some methods that work for later versions (look at the readme)
> > > > > 
> > > > > 
> > > > > okay but i upgrade it back when i dont need it?
> > > > 
> > > > 
> > > > if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)
> > > 
> > > 
> > > is it worth it?
> > 
> > 
> > not really, if you get caught you can be permanently banned from all school-issued devices, as it goes on your permanent record.
> 
> so is there a less risk way that you can tell me thats better then downgrading i am looking at the readme thing

the only other way is to use point blank, or idk.. killcurly

**Reply by Aka-but-nice** [reply]
> > > > > > > there are some methods that work for later versions (look at the readme)
> > > > > > 
> > > > > > 
> > > > > > okay but i upgrade it back when i dont need it?
> > > > > 
> > > > > 
> > > > > if you upgrade it, then the extensions will turn back on (if you turned them off after downgrading)
> > > > 
> > > > 
> > > > is it worth it?
> > > 
> > > 
> > > not really, if you get caught you can be permanently banned from all school-issued devices, as it goes on your permanent record.
> > 
> > 
> > so is there a less risk way that you can tell me thats better then downgrading i am looking at the readme thing
> 
> the only other way is to use point blank, or idk.. killcurly

Killcurly doesn't work for goguardian, goguardian stores its students information, so if you signout it still knows you're a student, and in its name "Kill" Curly(Securly) This is why it doesn't work for most blockers, it obviously works for securly
